ReleaseNotes.md

```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## New in Release 2.5.0
+Post-training Quantization:
+- Features:
+  - Added Post-training Quantization support for OpenVINO IR (OpenVINO backend).
+  - Added `"overflow_fix"` parameter (for `quantize(...)` & `quantize_with_accuracy_control(...)` methods) support & functionality. It improves accuracy for optimized model for affected devices.
```

Comment:
I would recommend to add some explanation why and how user should use it.

Reply:
Should I place the link into the original NNCF documentation? `docs/compression_algorithms/Quantization.md`